modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts

```diff
@@ -265,7 +265,7 @@ export class JournalEditArticlePage {
 		await this.fillTitle(title);
 
 		if (!(await this.page.getByText('Never Expire').isVisible())) {
-			await this.page.getByRole('link', {name: 'Schedule'}).click();
+			await this.page.getByRole('button', {name: 'Schedule'}).click();
 		}
 
```

Comment:
Use the new util [playwright/utils/openFieldset.ts](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/openFieldset.ts) for form and sidebar expandable panels.

You can view it in action [here](https://github.com/liferay-content-management/liferay-portal/pull/5924/commits/e6fa0e80e1faa4bf69622d1a7dc046e7ac2ebf9a) and [here](https://github.com/liferay-echo/liferay-portal/pull/16340/commits/f86c7c5479f075656e013bdde56fa7b494a33ccc).

Reply:
changed both for openFieldset! Thx @boton